www/index.html

```diff
@@ -203,5 +203,12 @@
     <script src="assets/js/vendor/mousetrap.min.js"></script>
     <script src="js/desktop_index.js"></script>
     <script type="text/javascript" src="js/vendor/tingle.js"></script>
+    <script>
+      const { remote } = require('electron');
+      const { i18n } = remote.require('./app');
```

Comment:
this technically a bad practice. future versions of electron will remove this. its fine for now.